src/main/java/springboot/webapp/usersmanager/controllers/UserController.java

```diff
@@ -0,0 +1,44 @@
+package springboot.webapp.usersmanager.controllers;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import springboot.webapp.usersmanager.services.UserService;
+import springboot.webapp.usersmanager.entities.User;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/users")
+public class UserController {
+
+    private final UserService userService;
+
+    @GetMapping
+    public ResponseEntity<List<User>> getAll() {
+        return ResponseEntity.ok(userService.getAll());
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<?> get(@PathVariable int id) {
+        return userService.get(id);
+    }
+
+    @PostMapping("/new")
+    public ResponseEntity<String> create(@RequestBody User user) {
+        return userService.save(user);
```

Comment:
Service shouldn't return "ResponseEntity". It is the controller area of responsibility